helpers/measures.ts

```diff
@@ -56,6 +56,27 @@ export function isDiffMetric(metricKey: string): boolean {
   return metricKey.indexOf('new_') === 0;
 }
 
+/*
+ * Conditional decimal count for QualityGate-impacting measures
+ * (e.g. Coverage %)
+ * Increase the number of decimals if the value is close to the threshold
+ * We count the precision (number of 0's, i.e. log10 and round down) needed to show the difference
+ * E.g. threshold 85, value 84.9993 -> delta = 0.0007, we need 4 decimals to see the difference
+ * otherwise rounding will make it look like they are equal.
+ */
+const DEFAULT_DECIMALS = 1;
+export function getDecimalsCount(value: number, threshold: number | undefined): number {
```

Comment:
```suggestion
export function getDecimalsCountToMakeThatPotentialyNotDistinctiveEnoughValueDistinctiveEnough(value: number, threshold: number | undefined): number {
```

Reply:
```suggestion
export function getMinDecimalsCountToBeDistinctFromThreshold(value: number, threshold: number | undefined): number {
```

Reply:
Arf, I mean, it will be minified anyway.

Reply:
I like the second one :+1: